riverdriver/riverdatabasesql/migration/main/005_migration_unique_client.down.sql

```diff
@@ -7,36 +7,41 @@
 DO
 $body$
 BEGIN
-    IF EXISTS (
-        SELECT *
-        FROM river_migration
-        WHERE line <> 'main'
-    ) THEN
-        RAISE EXCEPTION 'Found non-main migration lines in the database; version 005 migration is irreversible because it would result in loss of migration information.';
+    -- Tolerate users who may be using their own migration system rather than
+    -- River's. If they are, they will have skipped version 001 containing
+    -- `CREATE TABLE river_migration`, so this table won't exist.
+    IF (SELECT to_regclass('river_migration') IS NOT NULL) THEN
+        IF EXISTS (
+            SELECT *
+            FROM river_migration
+            WHERE line <> 'main'
+        ) THEN
+            RAISE EXCEPTION 'Found non-main migration lines in the database; version 005 migration is irreversible because it would result in loss of migration information.';
+        END IF;
+
+        ALTER TABLE river_migration
+            RENAME TO river_migration_old;
+
+        CREATE TABLE river_migration(
+            id bigserial PRIMARY KEY,
+            created_at timestamptz NOT NULL DEFAULT NOW(),
+            version bigint NOT NULL,
+            CONSTRAINT version CHECK (version >= 1)
+        );
+
+        CREATE UNIQUE INDEX ON river_migration USING btree(version);
+
+        INSERT INTO river_migration
+            (created_at, version)
+        SELECT created_at, version
+        FROM river_migration_old;
+
+        DROP TABLE river_migration_old;
     END IF;
 END;
 $body$
 LANGUAGE 'plpgsql'; 
 
-ALTER TABLE river_migration
-    RENAME TO river_migration_old;
-
-CREATE TABLE river_migration(
-  id bigserial PRIMARY KEY,
-  created_at timestamptz NOT NULL DEFAULT NOW(),
-  version bigint NOT NULL,
-  CONSTRAINT version CHECK (version >= 1)
-);
-
-CREATE UNIQUE INDEX ON river_migration USING btree(version);
-
-INSERT INTO river_migration
-    (created_at, version)
-SELECT created_at, version
-FROM river_migration_old;
-
-DROP TABLE river_migration_old;
```

Comment:
No actual changes here really, but everything moves up into the `IF` block/script above.